docs-guides/source/model-exporting.md

```diff
@@ -0,0 +1,120 @@
+```{eval-rst}
+.. index:: 
+    single: PyTorch; model exporting
+    single: model exporting
+    single: ExportedProgram
+```
+
+# Model Exporting
+
+The recommended way to generate ExportedProgram for your model is to use PyTorch's [torch.export.export](https://pytorch.org/docs/stable/export.html#torch.export.export). Exporting runs an example input tensor through your model, and captures the operations that are invoked as that input makes its way through the model's layers.
+
+```{admonition} Exporting Limitations
+
+The conversion from `torch.export` graph has been newly added to Core ML Tools 8.0.
+It is currently in beta state, in line with the export API status in PyTorch.
+As of Core ML Tools 8.0, representative models such as MobileBert, ResNet, ViT, [MobileNet](convert-a-torchvision-model-from-pytorch), [DeepLab](convert-a-pytorch-segmentation-model), [OpenELM](convert-openelm) can be converted, and the total PyTorch op translation test coverage is roughly ~60%. You can start trying the torch.export path on your models that are working with torch.jit.trace already, so as to gradually move them to the export path as PyTorch also [moves](https://github.com/pytorch/pytorch/issues/103841#issuecomment-1605017153) its support and development to that path over a period of time. In case you hit issues (e.g. models converted via export path are slower than the ones converted from jit.trace path), please report them on Github.
+
+Also, torch.export has limitations, see [PyTorch elaboration](https://pytorch.org/docs/stable/export.html#limitations-of-torch-export)
```

Comment:
instead of `[PyTorch elaboration]` , just say "see here[link]"

Reply:
Revised as suggested